config.yml

```diff
@@ -61,15 +59,13 @@ production:
   server:
     env:              production
   irc:
-    server:           irc.mozilla.org
-    nick:             taskcluster
-    userName:         taskcluster
-    realName:         Taskcluster Robot
```

Comment:
It looks like this is the key hunk, and the others are window-dressing?

Reply:
Just other stuff I ended up needing for r14y.